k9mail/src/main/java/com/fsck/k9/view/K9WebViewClient.java

```diff
@@ -21,34 +20,33 @@
 import android.webkit.WebViewClient;
 
 import com.fsck.k9.K9;
-import com.fsck.k9.mail.Body;
-import com.fsck.k9.mail.Multipart;
-import com.fsck.k9.mail.Part;
-import com.fsck.k9.mailstore.AttachmentViewInfo;
-import com.fsck.k9.message.extractors.AttachmentInfoExtractor;
+import com.fsck.k9.mailstore.AttachmentResourceProvider;
 
 
 /**
  * {@link WebViewClient} that intercepts requests for {@code cid:} URIs to load the respective body part.
  */
-public abstract class K9WebViewClient extends WebViewClient {
+abstract class K9WebViewClient extends WebViewClient {
     private static final String CID_SCHEME = "cid";
     private static final WebResourceResponse RESULT_DO_NOT_INTERCEPT = null;
     private static final WebResourceResponse RESULT_DUMMY_RESPONSE = new WebResourceResponse(null, null, null);
 
-    public static WebViewClient newInstance(Part part) {
+
+    // injected state
```

Comment:
I have a bunch of other classes where I distinguish fields between injected, persistent and transient state. I can get rid of this one, but I think there is some merit in having it for consistency